## 05-emdebian-wchar.dpatch by Neil Williams <codehelp@debian.org>
##
## DP: Ensure <wchar.h> is available for make_keys via gcc when cross-building
## DP: the rest of the package (otherwise it picks up the config.h for the
## DP: cross-build).
--- a/ncurses/tinfo/comp_hash.c
+++ b/ncurses/tinfo/comp_hash.c
@@ -39,6 +39,7 @@
  */
 
 #define USE_TERMLIB 1
+#include <wchar.h>
 #include <curses.priv.h>
 
 #include <tic.h>
--- a/ncurses/tinfo/make_keys.c
+++ b/ncurses/tinfo/make_keys.c
@@ -37,6 +37,7 @@
  */
 
 #define USE_TERMLIB 1
+#include <wchar.h>
 #include <curses.priv.h>
 
 MODULE_ID("$Id: make_keys.c,v 1.14 2008/08/03 21:57:22 tom Exp $")
